### ISSUE TITLE
Mismatch between README values.yaml and actual values.yaml

### ISSUE THREAD

**Issue by lindhe** [rancher/cluster-template-examples#21]
The example for `values.yaml` in the README doesn't match the contents of the actual [`values.yaml`](https://github.com/rancher/cluster-template-examples/blob/dc5c201c5415c430cdf4bcb6af975f9e516e970b/charts/values.yaml). If the example from the README is up-to-date, I think those should be merged into the real `values.yaml` file (as comments, if they are not reasonable default values).

https://github.com/rancher/cluster-template-examples/blob/dc5c201c5415c430cdf4bcb6af975f9e516e970b/README.md?plain=1#L7-L144

**Comment by lindhe** [comment]
It seems like `rancherValues` is unused, so that should be removed from the example.

This is resolved in #22.

**Comment by lindhe** [comment]
After #22, the example in README.md is almost identical to the contents of values.yaml. I think the examples should be removed from README.md, so we don't risk having out-of-date documentation again in the future.
